src/inventory.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file inventory.h
+ * @author Austin Arlitt
+ */
+
+#ifndef INVENTORY_H
+#define INVENTORY_H
+
+// includes TBD
+
+/**
+ * A class for managing the inventory of a movie rental store
+ */
+class Inventory {
+public:
+    Inventory();                        // default constructor, creates empty inventory
+    ~Inventory();                       // destructor
```

Comment:
Changing `Item *` to `std::shared_ptr<Item>` for member sets would eliminate the need for a destructor.